tests/test_logging.py

```
@@ -0,0 +1,321 @@
+    assert resp.status_code == 200
+
+    access = [r for r in capture.records if r.get("event") == "http_request"]
+    assert len(access) == 1
+    line = access[0]
+    assert line["method"] == "GET"
+    assert line["path"] == "/ping"
+    assert line["status"] == 200
+    assert isinstance(line["latency_ms"], (int, float))
+
+
+def test_response_carries_request_id_header(capture):
```

Ruff (0.15.13)

[warning] Unused function argument: `capture`

(ARG001)